src/main/webapp/populateData.js

```diff
@@ -364,6 +364,48 @@ function populateGo() {
       });
 }
 
+/**
+ * Populate Plan-mail panel with potential times to read the emails
+ */
+function populatePlanMail() {
+  const planContainer = document.querySelector('#plan');
+  fetch('/plan-mail')
+      .then((response) => {
+        // If response is a 403, user is not authenticated
+        if (response.status === 403) {
+          throw new AuthenticationError();
+        }
+        return response.json();
+      })
+      .then((planMailResponse) => {
+        // Display the potential times to create events
+        const numWordsContainer = document.querySelector('#word-count');
+        numWordsContainer.innerText = planMailResponse.wordCount;
+        const averageReplyContainer = document.querySelector('#average-reply');
+        averageReplyContainer.innerText = planMailResponse.averageReadingSpeed;
+        const timeNeededContainer = document.querySelector('#time-needed');
+        timeNeededContainer.innerText = planMailResponse.minutesToRead;
+        const intervalContainer = document.querySelector('#free-interval');
+        intervalContainer.innerHTML = '';
+        for (const index in planMailResponse.potentialMeetingTimes) {
+          if (typeof index == 'string') {
+            const liElement = document.createElement('li');
+            liElement.innerText =
+              planMailResponse.potentialMeetingTimes[index].start +
+              ' ' +
+              planMailResponse.potentialMeetingTimes[index].end;
```

Comment:
Oh I remember this, I am guessing the later branch replaces this?

Reply:
Yes, It is fixed in PR #150